docs/source/nlp/text_normalization.rst

```diff
@@ -42,18 +62,36 @@ and ``test.tsv``.
     DATE	2005	two thousand five
     PLAIN	and	<self>
     DATE	2008	two thousand eight
-    PUNCT	.	sil
+    PUNCT	.	<self>
     <eos>	<eos>
 
 
-An example script for generating a dataset in this format from the `Google text normalization dataset <https://www.kaggle.com/google-nlu/text-normalization>`_
-can be found at  `NeMo/examples/nlp/duplex_text_normalization/google_data_preprocessing.py <https://github.com/NVIDIA/NeMo/blob/main/examples/nlp/duplex_text_normalization/google_data_preprocessing.py>`__.
-Note that the script also does some preprocessing on the spoken forms of the URLs. For example,
-given the URL "Zimbio.com", the original expected spoken form in the Google dataset is
-"z_letter i_letter m_letter b_letter i_letter o_letter dot c_letter o_letter m_letter".
-However, our script will return a more concise output which is "zim bio dot com".
-
 More information about the Google text normalization dataset can be found in the paper `RNN Approaches to Text Normalization: A Challenge <https://arxiv.org/ftp/arxiv/papers/1611/1611.00068.pdf>`__ :cite:`nlp-textnorm-Sproat2016RNNAT`.
+The script for splitting the Google text normalization data files into `train`, `dev`, `test` can be found here: 
+`data/data_split.py <https://github.com/NVIDIA/NeMo/blob/main/examples/nlp/duplex_text_normalization/data/data_split.py>`__.
+
+Data preprocessing
+-----------
+
+Processing scripts can be found in the same folder. Right now we only provide scripts for English text normalization, see `data/en/data_preprocessing.py <https://github.com/NVIDIA/NeMo/blob/main/examples/nlp/duplex_text_normalization/data/en/data_preprocessing.py>`__.
+The details can be found at the top of the scripts.
+The purpose of the preprocessing scripts is to standardize the format in order to help with model training.
+For text normalization it is crucial to avoid unrecoverable errors, which are linguistically coherent and not semantic preserving. 
+We noticed that due to data scarcity the model struggles verbalizing long numbers correctly, so we changed the ground truth for long numbers to digit by digit verbalization.
+We also ignore certain semiotic classes from neural verbalization, e.g. `ELECTRONIC` or `WHITELIST`. Instead we label urls/email addresses and abbreviations as plain tokens, and handle it separately with WFST-based grammars, see :ref:`inference_text_normalization`.
```

Comment:
please add smth like. `WHITELIST`, i.e. VERABUTIM and LETTER in the original dataset